packages/wrangler/src/api/startDevWorker/types.ts

```diff
@@ -83,6 +84,8 @@ export interface StartDevWorkerInput {
 	/** The triggers which will cause the worker's exported default handlers to be called. */
 	triggers?: Trigger[];
 
+	tails?: CfTailConsumer[];
```

Comment:
I'd rather we call this prop `tailConsumers` as opposed to `tails` for both clarity, and most importantly consistency with the config key

Reply:
I went with tails for consistency with the `workerd` config, but can definitely change this.